### PR TITLE
Add Secrets Provider helm chart version 1.1.0

### PR DESCRIPTION
**Pull request by abrahamko**
Secrets Provider supports helm chart, so it is added to here too.

### CODE REVIEW

File: docs/index.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 entries:
   conjur-oss:
   - apiVersion: v1
-    created: "2020-06-18T16:52:36.877060888-04:00"
+    created: "2020-09-07T14:58:09.009245+03:00"
```
Comment (sigalsax) [inline docs/index.yaml:5]: @izgeri @abrahamko are these intentional changes or something that gets changed automatically?

**Comment by sigalsax** [comment]
@abrahamko LGTM! Let's wait until we push a release before merge. Thanks!

File: docs/index.yaml (hunk 1 above)
Comment (abrahamko) [reply docs/index.yaml:5]: All the changes in the index file are the result of running the `reindex.sh` script.

**Review by izgeri — APPROVED** [review]
LGTM

**Review by sigalsax — APPROVED** [review]
LGTM!

**Pull request merged**